lib/medera/slack/event.ex

```diff
@@ -38,6 +44,8 @@ defmodule Medera.Slack.Event do
   defp event_type(%{type: "hello"}),           do: :hello
   # when a user is setting/unsetting 'away'
   defp event_type(%{type: "presence_change"}), do: :presence_change
+  # user opens a direct message channel
+  defp event_type(%{type: "im_open"}),         do: :im_open
```

Comment:
Would this work or are you trying to whitelist these types? `defp event_type(%{type: type}), do: String.to_atom(type)`